runtime/runtime.go

```diff
@@ -25,6 +25,8 @@ type Interface interface {
 	JoinNodes(newNodesIPList []string) error
 	DeleteMasters(mastersIPList []string) error
 	DeleteNodes(nodesIPList []string) error
+	CopyFilesOnRoot(cluster *v1.Cluster) error
```

Comment:
Please make the returned error meaningful. 

When defining an interface, it would be much better to think every parameter carefully why we need it, how we make use of it.

This Interface definition seems to be a little bit casual. I think we can make it better.